snake.py

```diff
@@ -15,64 +17,87 @@
 win.border(0)
 win.nodelay(1)
 
-key = KEY_RIGHT                                                    # Initializing values
+# Initializing values
+key = KEY_RIGHT
 score = 0
 
-snake = [[4,10], [4,9], [4,8]]                                     # Initial snake co-ordinates
-food = [10,20]                                                     # First food co-ordinates
+# Initial snake co-ordinates
+snake = [[4, 10], [4, 9], [4, 8]]
+# First food co-ordinates
+food = [10, 20]
 
-win.addch(food[0], food[1], '*')                                   # Prints the food
+# Prints the food
+win.addch(food[0], food[1], '*')
 
-while key != 27:                                                   # While Esc key is not pressed
+# While Esc key is not pressed
+while key != 27:
     win.border(0)
-    win.addstr(0, 2, 'Score : ' + str(score) + ' ')                # Printing 'Score' and
-    win.addstr(0, 27, ' SNAKE ')                                   # 'SNAKE' strings
-    win.timeout(150 - (len(snake)/5 + len(snake)/10)%120)          # Increases the speed of Snake as its length increases
-    
-    prevKey = key                                                  # Previous key pressed
-    event = win.getch()
-    key = key if event == -1 else event 
+    # Printing 'Score' and
+    win.addstr(0, 2, 'Score : ' + str(score) + ' ')
+    # 'SNAKE' strings
+    win.addstr(0, 27, ' SNAKE ')
+    # Increases the speed of Snake as its length increases
+    win.timeout(150 - (len(snake)/5 + len(snake)/10) % 120)
 
+    # Previous key pressed
+    prevKey = key
+    event = win.getch()
+    key = key if event == -1 else event
 
-    if key == ord(' '):                                            # If SPACE BAR is pressed, wait for another
-        key = -1                                                   # one (Pause/Resume)
+    # If SPACE BAR is pressed, wait for another
+    if key == ord(' '):
+        # one (Pause/Resume)
+        key = -1
         while key != ord(' '):
             key = win.getch()
         key = prevKey
         continue
 
-    if key not in [KEY_LEFT, KEY_RIGHT, KEY_UP, KEY_DOWN, 27]:     # If an invalid key is pressed
+    # If an invalid key is pressed
+    if key not in [KEY_LEFT, KEY_RIGHT, KEY_UP, KEY_DOWN, 27]:
         key = prevKey
 
-    # Calculates the new coordinates of the head of the snake. NOTE: len(snake) increases.
+    # Calculates the new coordinates of the head
+    # of the snake. NOTE: len(snake) increases.
     # This is taken care of later at [1].
-    snake.insert(0, [snake[0][0] + (key == KEY_DOWN and 1) + (key == KEY_UP and -1), snake[0][1] + (key == KEY_LEFT and -1) + (key == KEY_RIGHT and 1)])
+    snake.insert(0, [snake[0][0] + (key == KEY_DOWN and 1) +
```

Comment:
Best practice is to put the + and - operators to the left of the arguments.  See [https://pep8.org/#break-before-or-after-binary-operator](url) for pretty examples. :)